apps/modernization-ui/src/pages/patient/profile/identification/identificationTransformer.ts

```diff
@@ -0,0 +1,31 @@
+import { PatientIdentification } from 'generated/graphql/schema';
+import { PatientIdentificationResult } from './useFindPatientProfileIdentifications';
+import { asLocalDate } from 'date';
+
+const internalized = (content: PatientIdentification): PatientIdentification | null => {
+    return (
+        content && {
+            ...content,
+            asOf: content.asOf && asLocalDate(content.asOf)
+        }
+    );
+};
+
+export const transform = (result: PatientIdentificationResult['findPatientProfile']): PatientIdentification[] => {
```

Comment:
issue:  This transformer is transforming the the array into the same type.  The other transformers are taking the types returned by graphql and transforming them into the types that our components understand.  The biggest gain is that we can transform the dates from `any` to `Date`.  This is the only way that sorting with work property for the date columns.